content/01.abstract.md

```diff
@@ -4,7 +4,7 @@ Researchers in ecology and evolutionary biology (EEB) are increasingly dependent
 With the growing role of data science in EEB, the use of efficient methods to collaborate, share, and reproduce code has become fundamental. 
 GitHub is an online, cloud-based service that can help researchers to track, organize, discuss, share, and collaborate on software and code.
 Despite these benefits, the use of GitHub by EEB researchers is not widespread due to the lack of domain-specific information and guidelines.
-To help EEB researchers adopt useful features from GitHub in their own workflows, we review thirteen practical ways to use the platform.
+To help EEB researchers adopt useful features from GitHub in their own workflows, we review twelve practical ways to use the platform.
```

Comment:
There are 12 sub-headings in the current version

Reply:
We need to change this all the way through - good spot!

Reply:
I did a find and replace, so I think I got them all.